src/codeu/chat/client/commandline/Chat.java

```diff
@@ -425,7 +428,34 @@ public void invoke(List<String> args) {
         System.out.println("    Exit the program.");
       }
     });
-
+    panel.register("u-authorize", new Panel.Command()
+    {
+      @Override
+      public void invoke(List<String> args) 
+      {
+        if (args.size() < 2) 
+        {
+      	  System.out.println("ERROR: Please provide enough parameters!!!");
```

Comment:
It would be more helpful if you provided a message reminding the user of the command format, such as,
"ERROR: Command does not follow the format m-authorize <user> <authority>"